include/unicorn/unicorn.h

```diff
@@ -933,6 +933,46 @@ uc_err uc_mem_write(uc_engine *uc, uint64_t address, const void *bytes,
 UNICORN_EXPORT
 uc_err uc_mem_read(uc_engine *uc, uint64_t address, void *bytes, size_t size);
 
+/*
+ Read a range of bytes in memory after mmu translation.
+
+ @uc:      handle returned by uc_open()
+ @address: starting virtual memory address of bytes to get.
+ @prot:    The access type for the tlb lookup
```

Comment:
As reviewed previously, if `@prot` is essential, please document a bit more here. Like how different protections could affect the behavior. For example, what does `uc_mem_read_virtual` with `prot = UC_PROT_WRITE` imply?